src/Ting/Repository/Hydrator.php

```diff
@@ -45,7 +46,7 @@ class Hydrator implements HydratorInterface
     protected $objectDatabase     = [];
     protected $objectSchema       = [];
     protected $unserializeAliases = [];
-    protected $alreadyManaged     = [];
+    protected ?WeakMap $alreadyManaged = null;
```

Comment:
no need to declare it nullable, as the instanciation is done in the constructor

Reply:
I declared it nullable since unit tests were failing. Tests failed because of  missing call to Hydrator class's constructor when inheritance was involved.
It's now properly fixed.